keyboards/playkbtw/pk60/keymaps/rfvizarra/readme.md

```diff
@@ -0,0 +1,11 @@
+# A personal layout for Play Keyboard60 based on the minila layout
```

Comment:
These are actually keymaps. A layout is the physical organization of keys on the board, what size they are, and where they go. A keymap is what each of those keys represent. Physical layout will not change unless you have a soldering iron and is represented by the `LAYOUT_minila` macro. 

```suggestion
# A personal keymap for Play Keyboard60 based on the minila layout
```